lang/php/Portfile

```diff
@@ -1343,9 +1343,8 @@ mysql.default_socket, mysqli.default_socket and pdo_mysql.default_socket\
 to the path to your MySQL server's socket file.
 
 For mysql5, use ${prefix}/var/run/mysql5/mysqld.sock
```

Comment:
The mention of `mysql5` can likely be removed as well: `mysql5` and `mysql51` are both for MySQL 5.1, but nobody ever removed one of the ports in favor of the other.
```suggestion
```